scripts/test-upgrade.sh

```diff
@@ -54,7 +54,15 @@ cd ${OA_DIR}/playbooks
 openstack-ansible lxc-containers-destroy.yml --limit repo_all
 openstack-ansible setup-hosts.yml --limit repo_all
 
-# TASK #2 TODO: This task should be moved somewhere towards the bottom once
+# TASK #3
+# https://github.com/rcbops/u-suk-dev/issues/392
+# Upgrade openstack-ansible
+pushd ${OA_DIR}
+export I_REALLY_KNOW_WHAT_I_AM_DOING=true
```

Comment:
This can be removed now, see https://github.com/openstack/openstack-ansible/commit/b472da537e22fd3c8afb8a3b9d70f4a57815035d